src/ape_test/provider.py

```diff
@@ -138,7 +141,16 @@ def send_call(self, txn: TransactionAPI, **kwargs) -> bytes:
 
     def send_transaction(self, txn: TransactionAPI) -> ReceiptAPI:
         try:
-            txn_hash = self.web3.eth.send_raw_transaction(txn.serialize_transaction())
+            if txn.signature:
+                txn_hash = self.web3.eth.send_raw_transaction(txn.serialize_transaction())
+            else:
+                if (
+                    txn.sender not in self.chain_manager.provider.web3.eth.accounts  # type: ignore # noqa
```

Comment:
Can you place the specific linting rules you need to disable here? Like `noqa[E501]`